include/nbl/macros.h

```diff
@@ -6,6 +6,7 @@
 #define __NBL_MACROS_H_INCLUDED__
 
 #include "nbl/core/decl/compile_config.h"
+#include <nbl/builtin/hlsl/macros.h>
 #include "assert.h"
 
 // basics

```

Comment:
you want to move out the basics to HLSL

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/519#discussion_r1331443218